src/rocksdb_options.rs

```diff
@@ -414,6 +421,61 @@ impl Drop for Statistics {
     }
 }
 
+pub struct WriteBufferManager {
+    pub inner: *mut DBWriteBufferManager,
```

Comment:
Why pub?

Reply:
It's used in `crocksdb_options_set_write_buffer_manager`, I'll change them to pub(crate) instead.